### PR TITLE
Tonyvanswet/ele 1479 make language selector work

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
https://linear.app/electricitymaps/issue/ELE-1479/make-language-selector-work
## Description

Select the language,  if i18 doesn't know default to english

### CODE REVIEW

File: web/src/features/map-controls/LanguageSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  const handleLanguageSelect = (languageKey: LanguageNamesKey) => {
+    i18n.changeLanguage(languageKey);
+    setSelectedLanguage(languageKey);
+    setLanguageSelectorOpen(false);
+  };
+  const languageOptions = languageKeys.map((key) => {
+    return (
+      <button
+        key={key}
+        onKeyDown={() => handleLanguageSelect(key)}
```
Comment (madsnedergaard) [inline web/src/features/map-controls/LanguageSelector.tsx:31]: Do we need both onClick and onKeyDown? 🤔

**Review by madsnedergaard — APPROVED** [review]
Nice!

File: web/src/features/map-controls/LanguageSelector.tsx (hunk 1 above)
Comment (VIKTORVAV99) [reply web/src/features/map-controls/LanguageSelector.tsx:31]: I suppose onKeyDown could enable keyboard navigation while onClick would enable mouse/trackpad/touch navigation. But then I think we would need a key handler to navigate through the list as well?

Comment (tonypls) [reply web/src/features/map-controls/LanguageSelector.tsx:31]: I wouldn't normally but eslint said it was good idea 🤷

**Pull request merged**